### PR TITLE
#1093 Fix templates tool aromatic attachment

### PR DESCRIPTION
**Pull request by karen-sarkisyan**
Fixes bug #1093  that I introduced in PR #1064

### CODE REVIEW

File: packages/ketcher-react/src/script/editor/tool/template.ts [hunk 1]
```diff
@@ -179,20 +179,16 @@ class TemplateTool {
       // calculate fragment center
       const xy0 = new Vec2()
       const bond = molecule.bonds.get(ci.id)
-      if (!bond) return
-      const frid = molecule.atoms.get(bond.begin)?.fragment
-      if (!frid) return
-      const frIds = molecule.getFragmentIds(frid)
+      const frid = molecule.atoms.get(bond?.begin as number)?.fragment
+      const frIds = molecule.getFragmentIds(frid as number)
```
Comment (AndreiMazol) [inline packages/ketcher-react/src/script/editor/tool/template.ts:183]: frid can be undefined here

**Review by AndreiMazol — CHANGES_REQUESTED** [review]
need to discuss

**Review by AndreiMazol — CHANGES_REQUESTED** [review]
need to  discuss: this case should be covered by aromaticFusing.ts

**Comment by karen-sarkisyan** [comment]
Closing PR because changes that introduced the bug have been reverted.

**Pull request closed without merging**